app/components/search.tsx

```diff
@@ -93,54 +97,73 @@ export default function Search({onSiteAnswersRef, openQuestionTitles, onSelect}:
-          onKeyDown={(e) => e.key === 'Enter' && searchFn(e.currentTarget.value)}
-        />
-        <MagnifyingGlass />
-      </label>
-      <AutoHeight>
-        <div className={`dropdown ${showResults && results.length > 0 ? '' : 'hidden'}`}>
-          <div>
-            {!tfFinishedLoadingRef.current && (
-              <i>Showing plain text search results while tensorflow is loading:</i>
-            )}
-            {showResults &&
-              results.map(({pageid, title, score}) => (
-                <ResultItem
-                  key={pageid}
-                  {...{
-                    pageid,
-                    title,
-                    score,
-                    model,
-                    onSelect: (...args) => {
-                      setHide(true)
-                      hideSearchResults()
-                      onSelect(...args)
-                    },
-                    isAlreadyOpen: openQuestionTitles.includes(title),
-                    setHide,
-                  }}
-                />
-              ))}
-          </div>
-          <AddQuestion
-            title={searchInputRef.current}
-            relatedQuestions={results.map(({title}) => title)}
-            onMouseDown={() => setHide(false)}
-            onMouseUp={() => setHide(true)}
+    <>
+      <div onFocus={() => setShowResults(true)} onBlur={handleBlur}>
+        <label className="searchbar">
+          <input
+            type="search"
+            name="searchbar"
+            placeholder="Search for more questions here..."
+            autoComplete="off"
+            onChange={(e) => handleChange(e.currentTarget.value)}
+            onKeyDown={(e) => e.key === 'Enter' && searchFn(e.currentTarget.value)}
           />
-        </div>
-      </AutoHeight>
-    </div>
+          <MagnifyingGlass />
+        </label>
+        <AutoHeight>
+          <div className={`dropdown ${showResults && results.length > 0 ? '' : 'hidden'}`}>
+            <div>
+              {!tfFinishedLoadingRef.current && (
+                <i>Showing plain text search results while tensorflow is loading:</i>
+              )}
+              {showResults &&
+                results.map(({pageid, title, score}) => (
+                  <ResultItem
+                    key={pageid}
+                    {...{
+                      pageid,
+                      title,
+                      score,
+                      model,
+                      onSelect: handleSelect,
+                      isAlreadyOpen: openQuestionTitles.includes(title),
+                      setHide,
+                    }}
+                  />
+                ))}
+            </div>
+            <button
+              className="result-item none-of-the-above"
+              onClick={() => setShowMore(true)}
+              onMouseDown={() => setHide(false)}
+              onMouseUp={() => setHide(true)}
+            >
+              Show more answers
```

Comment:
@plexish @ccstan99 This is displayed under the results for stuff that are live on the site - I'm hoping you'll come up with a better text

Reply:
Maybe "More options..."

Reply:
hmm.. 
> I'm asking something else

Maybe?

Reply:
changed to "I'm asking something else"